src/responses/src/search.rs

```diff
@@ -288,6 +289,26 @@ impl<T> Hit<T> {
     pub fn score(&self) -> Option<f32> {
         self.score.clone()
     }
+
+    /**
+    A reference to the [highlighted] snippets of the part(s) of the field(s)
+    matching the search query.
+
+    [highlighted]: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html
+    */
+    pub fn highlight(&self) -> Option<&Value> {
+        self.highlight.as_ref()
+    }
+
+    /**
+    A clone of the [highlighted] snippets of the part(s) of the field(s)
+    matching the search query.
+
+    [highlighted]: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html
+    */
+    pub fn into_highlight(&self) -> Option<Value> {
```

Comment:
Do we need this method? I think you could just call `response.highlight().cloned()` and get the same result.